media-tv/mythtv/mythtv-29.1-r2.ebuild

```diff
@@ -62,9 +62,6 @@ COMMON="
 		sys-fs/udisks:2
 	)
 	cec? ( dev-libs/libcec )
-	dvb? (
```

Comment:
Shouldn't this be removed from the IUSE list as well?

I think these commits look good, just the commit summaries need some renaming and they are a bit all over the place. Anyway I can edit those, just need an answer what would happen if dvb was removed from IUSE.

Reply:
Do not remove dvb use flag. The dvb use flag is passed to mythtv. The dependency is no longer needed as headers are in kernel now.